nest/02.nest.md

```diff
@@ -0,0 +1,99 @@
+# 2 nest
+In this chapter, we will detail how to use the command `nest`.
+As explained before, this command is mainly designed for ease of use.
+However, anyone can use it.
+
+## 2.1 nest commands
+### 2.1.1 Updating the list of available packages with `nest pull`
+Nest uses a local cache to store the list of available packages you can install.
+When you're installing a package, Nest will search for it in this cache.
+The command to update the cache containing the list of available packages, is `nest pull`.
+
+Its output looks like this:
+
+```
+$ nest pull
+1 pending transaction:
+
+    pull stable
+
+Would you like to apply this transaction? [Yes/no] 
+Pulling stable...
+Successfully pulled 1 repository
+```
+
+> Attention
+>
+> Nest won't be able to install packages that are not in your cache, so it's really important to keep your cache up-to-date as often as possible.
```

Comment:
> your cache

You could include a link to what this is.

Reply:
The cache is explained just at the start of the same section, I don't think it's necessary to link this to the same section it is in

Reply:
OK.